app/src/main/java/com/portfolio/tasky/globals/Constants.kt

```diff
@@ -0,0 +1,12 @@
+package com.portfolio.tasky.globals
+
+class Constants {
```

Comment:
I'd personally use more grouped classes for constants and not one global constants class. In the end you'll have different features and each will have its own set of constants. You already kinda did that by having different objects in this class, but to be able to have a clear separation of concerns, it doesn't work to have a global place for this